### PR TITLE
Adds system resources check to make sure minikube is started with minimum resources

### PR DESCRIPTION
**Pull request by bharathappali**
Signed-off-by: bharathappali <bharath.appali@gmail.com>

### CODE REVIEW

**Comment by bharathappali** [comment]
@dinogun Can I please have this reviewed?

File: minikube_demo_setup.sh [hunk 1]
```diff
@@ -44,6 +49,32 @@ function check_err() {
 	fi
 }
 
+# Prints the minimum system resources required to run the demo
+function print_min_resources() {
```
Comment (dinogun) [inline minikube_demo_setup.sh:53]: Please use tabs for this PR

File: minikube_demo_setup.sh [hunk 2]
```diff
@@ -44,6 +49,32 @@ function check_err() {
...
+  echo "       Minikube resource config needed for demo:"
+  echo "       CPUs=8, Memory=16384MB"
+}
+
+# Checks if the system which tries to run autotune is having minimum resources required
+function sys_cpu_mem_check() {
+  SYS_CPU=$(lscpu -p | grep -v '^#' | wc -l)
+  SYS_MEM=$(grep MemTotal /proc/meminfo | awk '{printf ("%.0f\n", $2/(1024))}')
+
+  if [ "$SYS_CPU" -lt "$MIN_CPU" ]; then
```
Comment (dinogun) [inline minikube_demo_setup.sh:63]: Best practice is to use quotes and brackets

```
 "${SYS_CPU}"      "${MIN_CPU}"
```
Please make this change everywhere

**Review by dinogun — CHANGES_REQUESTED** [review]
Minor changes needed

File: minikube_demo_setup.sh (hunk 1 above)
Comment (bharathappali) [reply minikube_demo_setup.sh:53]: @dinogun Thanks for the review I have made the changes to use Tabs. Thanks

File: minikube_demo_setup.sh (hunk 2 above)
Comment (bharathappali) [reply minikube_demo_setup.sh]: @dinogun I have made the changes

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**